en/mozillavpn.xlf

```diff
@@ -0,0 +1,505 @@
+      <context-group purpose="location"><context context-type="linenumber">41</context></context-group>
+    </trans-unit>
+    <trans-unit id="learnMore">
+      <source xml:space="preserve">Learn more</source>
+      <context-group purpose="location"><context context-type="linenumber">54</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/views/ViewMain.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="newVersionAvailable">
+      <source xml:space="preserve">New version is available.</source>
+      <context-group purpose="location"><context context-type="linenumber">28</context></context-group>
+    </trans-unit>
+    <trans-unit id="updateNow">
+      <source xml:space="preserve">Update now</source>
+      <context-group purpose="location"><context context-type="linenumber">30</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/views/ViewUpdate.qml</context><context context-type="linenumber">180</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/views/ViewUpdate.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="updateRecomended">
+      <source xml:space="preserve">Update recomended</source>
+      <context-group purpose="location"><context context-type="linenumber">24</context></context-group>
+    </trans-unit>
+    <trans-unit id="updateRecomended.description">
+      <source xml:space="preserve">Please update the app before you\ncontinue to use the VPN</source>
+      <context-group purpose="location"><context context-type="linenumber">26</context></context-group>
+    </trans-unit>
+    <trans-unit id="updateRequired">
+      <source xml:space="preserve">Update Required</source>
+      <context-group purpose="location"><context context-type="linenumber">58</context></context-group>
+    </trans-unit>
+    <trans-unit id="updateRequire.reason">
+      <source xml:space="preserve">We detected and fixed a serious bug.\nYou must update your app.</source>
+      <context-group purpose="location"><context context-type="linenumber">60</context></context-group>
+    </trans-unit>
+    <trans-unit id="updateConnectionInsecureWarning">
+      <source xml:space="preserve">Your connection will not be secure while you update.</source>
+      <context-group purpose="location"><context context-type="linenumber">172</context></context-group>
+    </trans-unit>
+    <trans-unit id="notNow">
+      <source xml:space="preserve">Not now</source>
+      <context-group purpose="location"><context context-type="linenumber">194</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="cpp" original="../src/connectiondataholder.cpp" source-language="en" target-language="en-US"><body>
+    <trans-unit id="unknown">
+      <source xml:space="preserve">Unknown</source>
```

Comment:
This needs a comment explaining what it refers to (as an adjective, it needs to be declined for gender).